frontend/web/components/metadata/AddMetadataToEntity.tsx

```diff
@@ -334,7 +335,35 @@ const MetadataRow: FC<MetadataRowType> = ({
       <Flex className='table-column'>{`${metadata?.name} ${
         metadata?.isRequiredFor ? '*' : ''
       }`}</Flex>
-      {metadata?.type !== 'bool' ? (
+      {metadata?.type === 'bool' ? (
+        <Flex className='flex-row'>
+          <Switch
+            checked={!!metadataValue}
+            onChange={() => {
+              setMetadataValue(!metadataValue)
+              setMetadataValueChanged(true)
+              saveMetadata()
+            }}
+          />
+        </Flex>
+      ) : metadata?.type === 'multiline_str' ? (
+        <Flex className='flex-row' style={{ minWidth: '300px' }}>
+          <InputGroup
+            textarea
```

Comment:
cant you just adjust this single prop rather than everything?

Reply:
Updated